compiler/rustc_borrowck/src/type_check/liveness/trace.rs

```diff
@@ -161,8 +161,16 @@ impl<'me, 'typeck, 'flow, 'tcx> LivenessResults<'me, 'typeck, 'flow, 'tcx> {
         }
     }
 
-    // Runs dropck for locals whose liveness isn't relevant. This is
-    // necessary to eagerly detect unbound recursion during drop glue computation.
+    /// Runs dropck for locals whose liveness isn't relevant. This is
+    /// necessary to eagerly detect unbound recursion during drop glue computation.
+    ///
+    /// These locals should not result in any liveness constraints given that they
+    /// should otherwise not be considered boring and get dropped. This is currently
```

Comment:
```suggestion
    /// should otherwise be considered boring and get dropped. This is currently
```

are they or are they not considered boring?